tests/cases/fourslash/inlayHintsInteractiveMultifileFunctionCalls.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="fourslash.ts" />
+
+// @Target: esnext
+// @module: nodenext
+
+// @Filename: aaa.mts
+////import { helperB } from "./bbb.mjs";
+////helperB("hello, world!");
+
+// @Filename: bbb.mts
+////import { helperC } from "./ccc.mjs";
+
+////export function helperB(bParam: string) {
+////    helperC(bParam);
+////}
+
+// @Filename: ccc.mts
+////export function helperC(cParam: string) {}
+
+goTo.file("./aaa.mts");
```

Comment:
Is there a way to execute the baseline test in multiple files using a single fourslash test?

Reply:
I think you can run multiple baseline commands and it should work - maybe I'm wrong.

Reply:
If not you might need to avoid baselining and just use inline data.

Reply:
Running multiple baseline commands didn't work.

> If not you might need to avoid baselining and just use inline data.

I didn't get that?

Reply:
Weird, I guess there is no `verifyInlayHints` function. The only thing that runs multiple baselines directly is

`/workspaces/TypeScript/tests/cases/fourslash/cancellationWhenfindingAllRefsOnDefinition.ts`

it uses the helper `baselineCommands`. You can also make `baselineInlayHints` take a set of ranges and work similarly internally.